qml/Main.qml

```diff
@@ -86,7 +86,7 @@ PodcastsPage {
             properties = {};
         }
 
-        if (replace != true) {
+        if (!replace) {
```

Comment:
I think I prefer 
`replace !== true`

Though `!replace` seems to be functionally the same it actually introduces/maintains the same unpredictable behavior you would have removed by replacing `!=` with `!==`